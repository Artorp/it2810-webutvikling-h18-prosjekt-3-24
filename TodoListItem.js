import React from 'react';
import { StyleSheet } from 'react-native';
import { Button, Icon, Text, View } from 'native-base';

export default class TodoListItem extends React.Component {
  render() {
    return (
      <View style={styles.todoListItem}>
        <Text style={styles.itemText}>{this.props.name}</Text>
        <Button iconRight rounded small info style={styles.itemButton}>
          <Text>Fjern</Text>
          <Icon name="trash" />
        </Button>
      </View>
    );
  }
}

const styles = StyleSheet.create({
  todoListItem: {
    width: '100%',
    display: 'flex',
    flexDirection: 'row',
    justifyContent: 'space-between',
  },
  itemText: {
    width: '75%',
  },
  itemButton: {
    width: '25%',
  },
});
