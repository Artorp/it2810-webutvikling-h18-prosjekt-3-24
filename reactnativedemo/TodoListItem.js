import React from "react";
import { ScrollView, StyleSheet, Text, View } from "react-native";

export default class TodoListItem extends React.Component {
  render() {
    return <Text style={styles.text}>Gjøremål</Text>;
  }
}

const styles = StyleSheet.create({
  text: {
    color: "white"
  }
});
