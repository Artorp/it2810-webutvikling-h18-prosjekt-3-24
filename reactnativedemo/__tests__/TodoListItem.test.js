import React from 'react';
import renderer from 'react-test-renderer';
import TodoListItem from '../TodoListItem';

test('blir tegnet riktig', () => {
  const tree = renderer.create(<TodoListItem />).toJSON();
  expect(tree).toMatchSnapshot();
});
