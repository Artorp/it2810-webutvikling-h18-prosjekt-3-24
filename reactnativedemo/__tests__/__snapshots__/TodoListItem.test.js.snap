// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`blir tegnet riktig 1`] = `
<Text
  accessible={true}
  allowFontScaling={true}
  ellipsizeMode="tail"
  style={
    Object {
      "color": "#000",
      "fontFamily": "System",
      "fontSize": 16,
    }
  }
  uppercase={false}
  virtual={undefined}
>
  Gjøremål
</Text>
`;
